make push-dev-images seems to fail on Ubuntu

Not sure why, could be related to my shell as well.

Getting this:

```export DOCKER_NAME=gordo-deploy;\
export DOCKER_IMAGE=gordo-deploy;\
./docker_push.sh
./docker_push.sh: 32: [[: not found
./docker_push.sh: 37: [[: not found
./docker_push.sh: 42: [[: not found
Must provide --username with --password-stdin
make: *** [Makefile:14: push-workflow-generator] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,7 @@ WORKFLOW_GENERATOR_IMG_NAME := gordo-deploy
 
 # Create the image capable of rendering argo workflow generator
 workflow-generator:
-	docker build . -f Dockerfile-GordoDeploy -t $(WORKFLOW_GENERATOR_IMG_NAME);\
+	docker build . -f Dockerfile-GordoDeploy -t $(WORKFLOW_GENERATOR_IMG_NAME)
 
 # Publish image to the currently logged in docker repo
 push-workflow-generator: workflow-generator

--- a/docker_push.sh
+++ b/docker_push.sh
@@ -1,3 +1,4 @@
+#!/bin/bash
 # Script to push an docker image to a docker registry. The docker image can
 # either exist and be provided in the env variable DOCKER_IMAGE, or it will be
 # built if DOCKER_IMAGE is empty and DOCKER_FILE is provided

